SELearning.Infrastructure.Tests/CommentRepositoryTests.cs

```diff
@@ -0,0 +1,162 @@
+using System;
+using Xunit;
+using Microsoft.EntityFrameworkCore;
+using SELearning.Core.Comment;
+using System.Threading.Tasks;
+using SELearning.Core;
+using System.Collections.Generic;
+
+namespace SELearning.Infrastructure.Tests
+{
+    public class CommentRepositoryTests
+    {
+        private readonly CommentRepository _repository;
+        private readonly CommentContext _context;
+
+        private static readonly Section section = new Section
+        {
+            Id = "1",
+            Title = "C#",
+            Description = "C# tools",
+            Content = new List<Content>()
+        };
+        private static readonly Content content = new Content
+        {
+            Author = "Sarah",
+            Section = section,
+            Id = 1,
+            Title = "Video on Entity Core",
+            Description = "Nice",
+            VideoLink = "www.hej.dk"
+        };
+        private IEnumerable<Comment> _comments = new List<Comment>()
+        {
+            new Comment { Author = "Amalie", Id = 1, Text = "Nice", Content = content },
+                new Comment { Author = "Albert", Id = 2, Text = "Cool but boring", Content = content },
+                new Comment { Author = "Paolo", Id = 3, Text = "This is a great video", Content = content },
+                new Comment { Author = "Rasmus", Id = 4, Text = "Very inappropriate", Content = content }
+        };
+
+        public CommentRepositoryTests()
+        {
+            //setting up the comment connection
+            var connection = new SqliteConnection("Filename=:memory:");
+            connection.Open();
+            var builder = new DbContextOptionsBuilder<CommentContext>();
+            builder.UseSqlite(connection);
+            _context = new CommentContext(builder.Options);
+            _context.Database.EnsureCreated();
+
+            _repository = new CommentRepository(_context);
+
+            section.Content.Add(content);
```

Comment:
There is a nullable thing warning here.

Reply:
This is because Asger has declared almost properties in Content nullable. I don't think this is the right solution, since eg. id, author, etc. can't be null.

Reply:
But i think Asger is the one to solve it, when he merges his code into master?